services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -205,6 +212,12 @@ const CMSUserRoutes = ({
                         path={RoutesRecord.SUBMISSIONS_SUMMARY}
                         element={<SubmissionSummary />}
                     />
+                    {showMCCRSRecordNumber && (
+                        <Route
+                            path={RoutesRecord.SUBMISSIONS_MCCRSID}
+                            element={<MccrsId />}
```

Comment:
This change caused a flicker of the 404 not found page before going to the MCCRSID page.

https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/4d149859-bf88-4ece-962c-7f9c5b3af696

Reply:
This might be a tricky fix since we use a hook for feature flags inside the route's component. I wouldn't be opposed to moving this into a bug ticket.

Thoughts? @macrael @haworku

Reply:
Ughhh! And also yes this shouldn't block this PR. To me the flicker is a follow on bug ticket (sure) or just can be wrapped into edit work as we finish out this epic. @pearl-truss happy to take a look at this -  just lmk

Reply:
I think an answer is to only display the blank page until flags have been loaded

Reply:
I haven't been able to reproduce this yet. I tried switching the flag on and off and loading the app in an incognito window. If there's anything else I can try, that would be helpful. And I can include a fix for this in the edit ticket.

Reply:
I think locally, it happens so fast that you might not see it. If you visit the [deployed app](https://d1q6zrnw4f4g7b.cloudfront.net/) for this PR you should be able to see it there

Reply:
On local, throttling network speeds might get the bug to show up.
![Screenshot 2023-10-24 at 10 41 12 PM](https://github.com/Enterprise-CMCS/managed-care-review/assets/98117700/d69b9be2-0397-424a-bf2f-b64f54ca6120)

Reply:
Got it, thanks Jason :)